# Define the goal
int32 desiredState
---
# Define the result
bool executed
---
bool feedback